dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/CamelJBangMain.java

```diff
@@ -29,6 +29,7 @@
     public static void run(String... args) {
         commandLine = new CommandLine(new CamelJBangMain())
                 .addSubcommand("run", new Run())
+                .addSubcommand("create-project", new CreateProject())
```

Comment:
I think it would be better to use two sub-commands so that we maintain the same pattern. I mean, it would be better to use `create project`. Look at how we do for the `init kamelet` feature.

Reply:
Thanks for the review, I made the requested changes and the project can be created with
```
jbang -Dcamel.jbang.version=3.15.0-SNAPSHOT CamelJBang@apache/camel create project TimerRoute.java test.yaml --quarkus-dependency=camel-quarkus-timer,camel-quarkus-log,camel-quarkus-yaml-dsl,camel-quarkus-http --name=TestProject
```
help output from create command is
```
➜ jbang -Dcamel.jbang.version=3.15.0-SNAPSHOT CamelJBang@apache/camel create -h
[jbang] Building jar...
Usage: CamelJBang create [-h] [COMMAND]
Creates Maven Project (use --help)
  -h, --help   Display the help and sub-commands
Commands:
  project  Creates Camel-Quarkus project
  ```